src/tools/handlers/handlers.test.ts

```
@@ -190,26 +323,39 @@ describe('listContentSourcesHandler', () => {
   });
 
-  it('should show source names', async () => {
+  it('should show status indicators', async () => {
     const result = await listContentSourcesHandler({}, context);
+    const text = result.content[0].text;
 
-    expect(result.content[0].text).toContain('Swift by Sundell');
-    expect(result.content[0].text).toContain('Antoine van der Lee');
-    expect(result.content[0].text).toContain('Nil Coalescing');
+    // Should have status indicators (enabled/disabled/needs setup)
+    expect(text).toMatch(/[✅⚙️⬜]/);
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)